app/helpers/quadicon_helper/decorator.rb

```diff
@@ -0,0 +1,12 @@
+module QuadiconHelper::Decorator
+  private
+
+  def status_img(item = nil)
+    case item.authentication_status
+    when "Invalid" then "100/x.png"
```

Comment:
can the "100" be not repeated/gotten from somewhere else? it might be changed tomorrow to 150 :) and this hardcoding will cause a regression

Reply:
This is question probably for @skateman since it's related to assets pipeline and it used to be in this exact format.

Reply:
@abonas we have the `100` everywhere and we want to keep it consistent. Eventually, we're going to replace all these things with fonticons (that aren't yet supported by quadicons).